### ISSUE TITLE
variable 'store' set but not used

### ISSUE THREAD

**Issue by boldt** [icing/mod_md#63]
Following the [Ubuntu installation guide](https://github.com/icing/mod_md/wiki/2.4.x-Installation#ubuntu), I ran the following commands:

```
mod_md > ./configure --with-apxs=/usr/bin/apxs --enable-werror
mod_md > make
```

I get the following error:

```
mod_md.c:737:17: error: variable 'store' set but not used [-Werror=unused-but-set-variable]
     md_store_t *store;
```

Workaround: I removed the option `--enable-werror`

**Comment by icing** [comment]
Tststs.

Thanks for the report. I build it regularly with clang and werror, seems I miss this warning there. Do you also get this on the current master?

**Comment by boldt** [comment]
I downloaded the release 1.0.2. Current master I have to check. I'll let you know.

**Comment by icing** [comment]
Thanks a lot. Fixed in v1.0.3.

**Comment by boldt** [comment]
I can confirm the fix.
